### PR TITLE
Add estimate endpoint for CNARW

### PR DESCRIPTION
**Pull request by orazve**
Added estimate endpoint for CNARW algorithm

### CODE REVIEW

**Comment by Mats-SX** [comment]
The failing test is already a problem; not introduced by this PR.

**Review by Mats-SX — APPROVED** [review]
LGTM

**Review by Mats-SX — APPROVED** [review]
LGTM

File: doc/sphinx/source/algorithms.rst [hunk 1]
```diff
@@ -92,7 +92,7 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
 
     Constructs a random subgraph based on random walks with restarts.
 
-.. py:function:: gds.alpha.graph.sample.cnarw(graph_name: str, from_G: Graph, **config: Any)
+.. py:function:: gds.graph.sample.cnarw(graph_name: str, from_G: Graph, **config: Any)
```
Comment (adamnsch) [inline doc/sphinx/source/algorithms.rst:95]: Do we not also need to add the new estimate endpoint here? :thinking:

Comment (adamnsch) [reply doc/sphinx/source/algorithms.rst:95]: We should have tests that catch this

Comment (orazve) [reply doc/sphinx/source/algorithms.rst:95]: Looks like its mentioning was missed in `example_server_endpoints.py` file. Thank you for finding, I will fix

**Pull request merged**